src/lib/components/workflow/filter-search/text-filter.svelte

```diff
@@ -22,6 +22,8 @@
 </script>
 
 <Input
+  label={translate('search')}
```

Comment:
Similarly, how about using the attribute here so it reflects what should go in the input?

```suggestion
  label={$filter.attribute)}
```

Reply:
Will use the attribute same as list-filter